src/main/java/com/iota/iri/TransactionValidator.java

```diff
@@ -69,10 +73,20 @@ public int getMinWeightMagnitude() {
         return MIN_WEIGHT_MAGNITUDE;
     }
 
-    private static void runValidation(TransactionViewModel transactionViewModel, final int minWeightMagnitude) {
+    private static boolean invalidTimestamp(TransactionViewModel transactionViewModel) {
```

Comment:
by convention it is better to call this: `hasInvalidTimeStamp`
See: http://geosoft.no/development/javastyle.html